attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto

```diff
@@ -136,6 +136,7 @@ message AttributeMetadataFilter {
   repeated AttributeScope scope = 2 [deprecated = true]; // Use scope_string instead
   repeated string key = 4;
   repeated string scope_string = 5;
+  bool internal = 6;
```

Comment:
discussed offline to have this as

```suggestion
  optional bool internal = 6;
```

because, for the default filter instance, we would want all the attributes, and not just the external attributes